index.rst

```diff
@@ -100,73 +110,268 @@ Data Sets Types and Goals
+  - ideally, data will be taken over multiple seasons to enable clear separation of templates from the science images
+  - datasets sampling a range of timescales (hours, days, ... years) provide the most complete look at the real transient and variable population
+  - datasets with multiple filters will aid in understanding our DCR performance
+  - substantial dithering or field overlaps will allow us to test our ability to piece together templates from multiple images (some transient surveys, such as HiTS, PTF, and ZTF, use a strict field grid)
+  - there is a balance to be struck between using datasets that have been extensively mined scientifically by the survey times as opposed to datasets that have not been exploited completely.  If published catalogs of variables, transients, and/or asteroids exist, they will aid in false-positive discrimination and speed QA work.  On the other hand well-mined datasets may be less motivating to work on, particularly for those outside LSST DM.
+  - LSST-like cadences to test Solar System Orbit algorithms
+
+CI
+==
+1. DECam HiTS
+    - A subset of data intended for CI AP testing (with Blind15A_40 and Blind15A_42) is in
+      https://github.com/lsst/ap_verify_ci_hits2015
+    This subset is only 3 visits and 2 CCDs per visit.
+    Presently (2018-08-15) the data are on a branch, not yet merged to master.
+
+SMALL
+=====
+1. DECam HiTS
+    - See https://dmtn-039.lsst.io/
+    - Available on lsst-dev in `/datasets/decam/_internal/hits`
+    - Total of 2269 images available.
+    - up to 14 DECam fields taken over two seasons, or a larger number (40-50) of single season-only ; 4-5 epochs per night in one band (g) over a week
+    - Essentially only g-band, as there are only a few r-band images available.  This would not then actually satisfy the 2-band MEDIUM color requirement outlined above.
+    - Blind15A_26, Blind15A_40, and Blind15A_42 have been selected for AP testing in
+      https://github.com/lsst/ap_verify_hits2015
+
+MEDIUM
+======
+1. HSC SSP PDR1+PDR2
+    - Planned work to build templates from PDR1 and then run subtractions from the new data in PDR2 from later years.
+    https://jira.lsstcorp.org/browse/DM-20559
+    https://jira.lsstcorp.org/browse/DM-20560
+
+It's less clear that it's important to do active regular testing of DIA on LARGE datasets.  MEDIUM should be sufficient to characterize the key science performance goals.
+
+
+AP Candidate Additional Datasets
+================================
+1. DECam DES SN fields
+    - 8 shallow SN fields, 2 deep SN fields
+    - griz observation sequences obtained ~ weekly
+    - deep fields have multiple exposures in one field in the same filter each night, with other filters other nights; shallow fields have a single griz sequence in one night.  Former is more LSST-like.
+    - Raw data are public
+    - 10 fields from 2014 (DES Y2) in field SN-X3.
+    - g (no particular reason for this choice)
```

Comment:
I don't think the g + visit numbers match what's on lsst-dev: I see griz, and a much larger number of visits.   I think @ericmorganson put this together and can summarize it.

Reply:
I was specifically referring to `/datasets/des_sn/repo_Y2`.  I have updated the text accordingly.

But more generally, @ericmorganson, do you have some high-level description text that she supplement the current barebones presentation here of this subset of DES SN fields?